plugwise/data.py

```diff
@@ -204,7 +204,7 @@ def _device_data_climate(self, device: DeviceData, data: DeviceData) -> None:
         # Operation modes: auto, heat, heat_cool, cool and off
         data["mode"] = "auto"
         self._count += 1
-        if sel_schedule == NONE:
+        if sel_schedule in (NONE, OFF):
```

Comment:
I would suggest we keep using `lists` (i.e. `[]`) and not `set`s (i.e. `()`)

Reply:
See my comment on Discord.

Reply:
Further changes implemented as indicated on Discord.